src/app/query/common/crypto-asset.model.ts

```diff
@@ -0,0 +1,52 @@
+import type {
+  Blockchains,
+  Brc20CryptoAssetInfo,
+  BtcCryptoAssetBalance,
+  BtcCryptoAssetInfo,
+  CryptoAssetBalance,
+  CryptoAssetType,
+  MarketData,
+  Sip010CryptoAssetInfo,
+  StxCryptoAssetBalance,
+  StxCryptoAssetInfo,
+} from '@leather-wallet/models';
+
+interface CryptoAsset {
```

Comment:
RE the conversation we had on the other PR, this I gather is the more broad, everything type at query level. 

I just wonder on naming. Does this type describe the `CryptoAsset`? To me this means means much more. User balance info, market data etc. Much more dynamic changing data than the name suggests.

/gpt what's a good name for a Typescript interface that contains information about a cryptocurrency, including the current price information, a given users balance